ui/widgets/nuxeo-user-suggestion.js

```diff
@@ -243,27 +243,26 @@ import './nuxeo-user-group-formatter.js';
     }
 
     _selectionFormatter(value) {
-      return escapeHTML(
-        ((item) => {
-          if (item) {
-            if (
-              item['entity-type'] === 'user' &&
-              item.properties &&
-              item.properties.firstName &&
-              item.properties.lastName
-            ) {
-              return `${item.properties.firstName} ${item.properties.lastName}`;
-            }
-            if (item['entity-type'] === 'group') {
-              return item.grouplabel ? item.grouplabel : item.groupname;
-            }
-            if (item.displayLabel) {
-              return item.displayLabel;
-            }
-            return `<span class="preserve-white-space">${item.id ? item.id : item})</span>`;
+      const content = ((item) => {
```

Comment:
Why do we need this be a function just to call it back with the parameter of the parent function?

Reply:
You were seeing a draft :P